package.json

```diff
@@ -76,7 +76,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "rimraf": "^5.0.5",
```

Comment:
I think this is not needed because tsup has a clean option which is already set to true. I know this has nothing to do with the PR, they're just a small boyscout rule thing